apps/common_files/ood_session.sh.erb

```diff
@@ -0,0 +1,15 @@
+export OOD_SESSION_STAGED_ROOT=<%= session.staged_root %>
+
+# timeout for apps that need to wait for a server response
```

Comment:
this shouldn't be here?
The `wait_server_start` already has a fallback to 180 seconds which seems to be the good default unless special cases?

Reply:
180 seconds seems a bit tight, this includes loading modules (which is slow), starting processes and connecting... I'll test what happens with that default

Reply:
quick test done, MATLAB definitely needs more time and RStudio can also benefit from that. Fixed in https://github.com/vub-hpc/ondemand-vub/pull/38/commits/8e40b456b7f2d0540e69d0fc3d12912140a86fd7